pkg/helpers/autoimport.go

```diff
@@ -102,12 +98,6 @@ type ImportHelper struct {
 	log            logr.Logger
 
 	generateClientHolderFunc GenerateClientHolderFunc
-	applyResourcesFunc       ApplyResourcesFunc
-}
-
-func (i *ImportHelper) WithApplyResourcesFunc(f ApplyResourcesFunc) *ImportHelper {
```

Comment:
If I remember correctly, this was added for the unit tests, want to confirm if removing this will impact the tests?